File: backend/src/main/scala/kpn/api/common/subset/SubsetOrphanNodesPage.scala
package kpn.api.common.subset

import kpn.api.common.OrphanNodeInfo
import kpn.api.common.TimeInfo

case class SubsetOrphanNodesPage(
  timeInfo: TimeInfo,
  subsetInfo: SubsetInfo,
  nodes: Seq[OrphanNodeInfo]
)

File: backend/api/src/main/java/kpn/api/common/subset/SubsetOrphanNodesPage.java
package kpn.api.common.subset;

import kpn.api.common.OrphanNodeInfo;
import kpn.api.common.TimeInfo;
import kpn.api.common.subset.SubsetInfo;

import com.google.common.collect.ImmutableList;

public record SubsetOrphanNodesPage(
  TimeInfo timeInfo,
  SubsetInfo subsetInfo,
  ImmutableList<OrphanNodeInfo> nodes
) {}
